src/processor.ts

```diff
@@ -23,8 +23,8 @@ export interface EvmIndexerOptions {
   dbOptions: TypeormDatabaseOptions;
   finalityConfirmation: number;
   network: consts.NETWORK;
-  rentalFactoryAddress: string;
-  rentalManagerAddress: string;
+  createAddress: string;
+  stopAddress: string;
```

Comment:
Might be me, I find these terms confusing. These are, like in the previous code, protocol-related contracts right? Can we make that explicit somehow or do you think this is fine?

Reply:
those are legit names of the contracts